isbg.py

```diff
@@ -304,13 +304,19 @@ def dehexof(x):
     m = md5()
     m.update(imaphost)
     m.update(imapuser)
+    if imapproxyuser is None:
+        m.update('')
```

Comment:
I thought of it, but chose against for some reason... which can't readily remember now.
"Morning is smarter than the evening" as a saying goes; at least for backwards-compatibility with existing saved files, you're right - better not change the formula at all. :)